src/components/ChatPlaylistCard/ChatPlaylistCard.css

```diff
@@ -0,0 +1,28 @@
+.ChatPlaylistCard {
```

Comment:
consider using css modules (e.g. what the Home component does). That'll avoid the need to prefix your class names